src/liberty/devCommands.ts

```diff
@@ -399,7 +398,7 @@ export async function customDevMode(libProject?: LibertyProject | undefined, par
             if (customCommand !== undefined) {
                 // save command
                 customCommand = customCommand.trim();
-                if ( customCommand.length > 0 ) {
+                if (customCommand.length > 0) {
```

Comment:
Please do not do any formatting changes to code that is not added or modified by you for this PR.